tests/testsuite/bad_config.rs

```diff
@@ -6,6 +6,8 @@ use cargo_test_support::registry::{self, Package};
 use cargo_test_support::str;
 use cargo_test_support::{basic_manifest, project, rustc_host};
 
+use crate::utils::ext::CargoProjectExt;
```

Comment:
Should we put this in a prelude and maybe re-export `cargo_test_support::prelude::*` in it?

Reply:
That could even be done as a refactor commit before any of the other changes

Reply:
Also, we should probably check how https://doc.crates.io/contrib/tests/writing.html needs to be updated